article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -120,14 +119,19 @@ object ArticlePicker {
   }
 
   def getTier(page: PageWithStoryPackage)(implicit request: RequestHeader): RenderType = {
+
     val whitelistFeatures = featureWhitelist(page, request)
     val isEnabled = conf.switches.Switches.DotcomRendering.isSwitchedOn
-    val isBetaUser = ActiveExperiments.isParticipating(DotcomRenderingBeta)
+    val isAdFree = ArticlePageChecks.isAdFree(page, request)
+    val isCommercialBetaUser = ActiveExperiments.isParticipating(DotcomRenderingAdvertisements)
+
+    // add free pages always go through DCR provided it's turned on and we support its article features.
+    // pages with commercial aspects require the request to go through the DotcomRenderingAdvertisements abtest
 
-    val tier = if ((dcrCouldRender(page, request) && isEnabled && isBetaUser && !request.forceDCROff) || request.forceDCR) RemoteRender else LocalRenderArticle
+    val tier = if ((dcrCouldRender(page, request) && isEnabled && (isAdFree || isCommercialBetaUser) && !request.forceDCROff) || request.forceDCR) RemoteRender else LocalRenderArticle
```

Comment:
So instead of swapping the tests, we're graduating the old test to full fledged? Happy with that, just want to make sure I understand properly!

And for my brain we can segment the test audience pageviews in the data lake with:

- experiences: dotcom-rendring
- abtest: DotcomRenderingAdvertisements is variant

*But* if you're adFree AND in the AB test, it'll still trigger the test? Do we need to exclude `isAdFree` users from the test? Or log and filter in the data lake?

cc @nicl

Reply:
Yeah it's  a bit of a mind bender and I've made some assumptions about our goals for this (which may be wrong, or we never came to a concrete conclusion on!).

After this pr, all articles that are ad free will be rendered on dcr if we support it, so we would consider that test 'ended'.

From after this is merged, we are from that point interested in comparing the performance of commercial pages under DCR vs commercial pages under Frontend; 

I guess this should be equivalent to comparing these two sets of page views:

* DotcomRenderingAdvertisements is variant + experience is dotcom-rendering
* DotcomRenderingAdvertisements is control + experience is web

There's sort of a limitation in that the first group will include ad free pages and the second group won't (because if it was ad free, the experience would never be web...)

Reply:
> There's sort of a limitation in that the first group will include ad free pages and the second group won't

Yeah, that's my concern on that one, purely because that could skew things like perf data - DCR average will be higher because ad-free is quicker?

Looking in the pageviews table though in the data lake there is the boolean field `ads_loaded` - we could ask D&I to filter down on that to proxy OR we could add to the experiences if we know it's an ad free request (experiences is a Set). I'll ask D&I.

Reply:
I think the only way to make it totally pure as a test would be to exclude ad-free pages from going to DCR, but that seems a bit silly to de-support everything we've done so far.

Reply:
Spoke to D&I, they recognise adfree as users in a membershiptier so they're happy to filter that way:

- Is in A/B test variant
	- Is in experiences === dotcom-rendering
		- Is membershiptier = (whatever is adfree)

That should get us closer to measuring what we want to measure, and I think we have it all going through already.

Reply:
OR actually,  if we said:

 "if you're in the commercial test, you never get ad-free dcr pages"

then I guess we'd be sure that any page views in the DotcomRenderingAdvertisements test would be commercial.

Reply:
@MatthewJWalls @gtrufitt I'm happy with this approach. I'm don't think we've done enough to prove DCR is better quite yet (on the performance side) but agree it feels good to try and keep as much traffic on the platform in the meantime - the stats so far have been at least equivalent from D+I and it helps us gain confidence in the system in terms of requests and more eyes/bug reports etc.